app/views/spam2/_revisions.html.erb

```diff
@@ -0,0 +1,68 @@
+<script>
+$(document).ready(function() {
```

Comment:
If I am not mistaken we have this exact same block in the `app/views/spam2/_comments.html.erb` file. This needs to be refactored. If there is a bug in one block, we would need to copy-paste it into another block which is not very effective and can cause more even more bugs.

Reply:
Both of them have different IDs and tables. 
` $("#comment_table").DataTable({` is in _comments.html.erb 
` $("#revision_table").DataTable({` is in _comments.html.erb

Reply:
You could create a function to encapsulate the block and pass an ID as an argument, right?

Reply:
Done!